Cargo.lock

```diff
@@ -45,9 +36,9 @@ dependencies = [
 
 [[package]]
 name = "aes"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Please don't do `cargo update` on this repo. We have too many dependencies to review this update meaningfully.
If you want to resolve `Cargo.lock` conflicts, do `git checkout origin/master Cargo.lock` followed by `cargo check -p polkadot-primitives`. 
If you want to update a specific dependency, do `cargo update -p <dep>`, if. all of the substrate, do `cargo update -p sp-io`.